Add release manual to readme (#1242)
nextcloud/nextcloud-vue: README.md

    - In the repository of an app do `npm link @nextcloud/vue` (you need to re-link any time you do `npm ci` in the app)
4. Then build the app with: `npm run build` (or watch for changes with `npm run watch`)

## Releasing a new version

1. Checkout latest master

> 2. Bump version in `package.json` and update `package-lock.json`

Reviewer: 🚨🚨🚨
```suggestion
2. use `npm version` to bump the version number
```

Author: ah that updates package.json?

Reviewer: Yes, it also automatically commits it and creates a git tag
